src/utilities/templatetags/utility_tags.py

```diff
@@ -58,10 +58,22 @@ def group_name():
 
 @register.inclusion_tag('utilities/list_of_links.html')
 def menu_list():
-    links = MenuItem.objects.filter(visible=True)
+    links = MenuItem.objects.get_main_menu_items().get_visible_items()
     return {'links': links}
 
 
+@register.inclusion_tag('utilities/side_menu_items.html', takes_context=True)
+def side_menu_list(context):
+    side_menu_items = MenuItem.objects.get_side_menu_items().get_visible_items()
+    return {'side_menu_items': side_menu_items, 'request': context.request}
+
+
+@register.inclusion_tag('utilities/side_menu_items_navbar.html', takes_context=True)
+def side_menu_list_navbar(context):
+    side_menu_items = MenuItem.objects.get_side_menu_items().get_visible_items()
+    return {'side_menu_items': side_menu_items, 'request': context.request if 'request' in context else None}
```

Comment:
These methods don't have any doc strings. What's the purpose behind this check for `request`?

Reply:
Docstrings should make it more clear what the difference is between these two tags.